### PR TITLE
chore(dependencies): replace babel-preset-latest with babel-preset-env

### PR DESCRIPTION
**Pull request by mgmcdermott**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["latest", "stage-2"]
+  "presets": ["env", "stage-2"]
```
Comment (kentcdodds) [inline .babelrc]: Should we need to configure `env`? Considering this package only supports Node v4, we could probably do something like [this](https://github.com/kentcdodds/nps-utils/blob/master/.babelrc#L3-L7)

**Comment by kentcdodds** [comment]
Would you like to [add yourself](../blob/master/CONTRIBUTING.md#add-yourself-as-a-contributor) to the [contributors table](../#contributors)?

File: .babelrc (hunk 1 above)
Comment (mgmcdermott) [inline .babelrc]: I agree, I was going to bring this up. I'll add a target of node 4.

**Comment by mgmcdermott** [comment]
Added myself to the contributors as well.

File: .all-contributorsrc [hunk 2]
```diff
@@ -113,6 +113,13 @@
       "contributions": [
         "tool"
       ]
+    },
+    {
+      "login": "mgmcdermott",
+      "name": "Michael McDermott",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/8161781?v=3",
+      "profile": "http://mgmcdermott.com",
+      "contributions": []
```
Comment (kentcdodds) [inline .all-contributorsrc]: Could you add `"code"` to this? And then run `npm start gen` to regenerate the `README.md`

File: package.json [hunk 3]
```diff
@@ -49,6 +49,9 @@
     "strip-indent": "^2.0.0",
     "validate-commit-msg": "^2.8.2"
   },
+  "engines": {
```
Comment (kentcdodds) [inline package.json]: Awesome :+1:

File: .all-contributorsrc [hunk 4]
```diff
@@ -113,6 +113,15 @@
...
         "tool"
       ]
+    },
+    {
+      "login": "mgmcdermott",
+      "name": "Michael McDermott",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/8161781?v=3",
+      "profile": "http://mgmcdermott.com",
+      "contributions": [
+        "code"
```
Comment (kentcdodds) [inline .all-contributorsrc]: Perfect

**Pull request merged**